garbage-collect/vendor/github.com/openfaas/openfaas-cloud/sdk/service.go

```diff
@@ -8,3 +8,17 @@ import (
 func FormatServiceName(owner, functionName string) string {
 	return fmt.Sprintf("%s-%s", strings.ToLower(owner), functionName)
 }
+
+func CreateServiceURL(URL, suffix string) string {
```

Comment:
Let's add a test for backwards compatibility where people can call this method more than once without getting the suffix added twice.

```
http://gateway.openfaas:8080 = fn(http://gateway:8080, openfaas)
http://gateway.openfaas:8080 = fn(http://gateway. openfaas:8080, openfaas)
```